src/ActionBar/ActionBar.test.js

```diff
@@ -6,7 +6,7 @@ import {
     ActionBarActions,
     ActionBarBack,
     ActionBarHeader
-} from './ActionBar';
+} from './';
```

Comment:
These are being imported from the auto generated `index` file in the component directory. Let me know if I should import directly from the component files.